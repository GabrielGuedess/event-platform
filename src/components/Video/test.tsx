import { render, screen } from '@testing-library/react';

import { Video } from '.';

describe('<Video />', () => {
  it('should render the heading', () => {
    const { container } = render(<Video />);

    expect(screen.getByText(/video/i)).toBeInTheDocument();

    expect(container.firstChild).toMatchSnapshot();
  });
});
