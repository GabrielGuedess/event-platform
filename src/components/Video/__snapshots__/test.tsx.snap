// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Video /> should render the heading 1`] = `
<div
  class="flex-1"
>
  Video
</div>
`;
